addons/sourcemod/scripting/vsh/abilities/ability_rage_ghost.sp

```diff
@@ -328,6 +438,16 @@ methodmap CRageGhost < SaxtonHaleBase
 			buttons &= ~IN_ATTACK;
 	}
 	
+	public void OnPlayerKilled(Event event)
+	{
+		//Purely cosmetic effect, but let's add a cool little icon for killing with the rage
+		if (g_bGhostEnable[this.iClient])
+		{
+			event.SetString("weapon_logclassname", "purgatory");
+			event.SetString("weapon", "purgatory");
+		}
+	}
```

Comment:
This need more checks, cases when player suicides while boss still credited (killbind, fall dmg or rocket jump at last moment). `inflictor_entindex` in `player_death` event may help.

Reply:
I had left it unchecked on purpose, as I thought that since the boss dealt damage to them before suiciding, it'd be worthy of showing the kill icon in these cases, though now that you say that, it does make more sense not to show it that way.